.github/workflows/sonar.yaml

```diff
@@ -70,30 +70,39 @@ jobs:
           github_repo_name="$(echo $GITHUB_REPOSITORY | cut -d'/' -f2)"
           echo "github_repo_name=$github_repo_name" >> $GITHUB_ENV
 
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          role-to-assume: "arn:aws:iam::${{ env.accountid }}:role/cdc-github-${{ env.github_repo_name }}-${{ env.deployment_env }}-role"
-          aws-region: us-east-1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
 
-      - name: Login to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v4
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
 
-      - name: Login to ECR docker registry
-        uses: docker/login-action@v2
+      - name: Build test dataingestion mssql image
```

Comment:
@mpeels what do you think about this strategy, just build the di-mssql image from source and skip the step of downloading a pre-built one from ECR.  It will add an extra min of runtime, but I think it should go a little faster on subsequent runs since we're caching the docker layers

Reply:
The only concern i would have would be the additional time it would take. An extra minute isn't bad. If it turns out to be an extra 5 minutes. I might prefer pulling the image from somewhere.